src/app/components/list/list.js

```diff
@@ -95,12 +95,11 @@ listControllers.controller('ListCtrl', ['$scope', '$routeParams', '$location', '
   $scope.isOwner = false;
   $scope.isFavorite = false;
 
-  $scope.$on('user-service-ready', function() { 
+  $scope.$on('user-service-ready', function() {
     $scope.list = List.get({'listId': $routeParams.list});
     var listCallback = function () {
       var req = [];
       req[$scope.list.type + 's.list'] = $scope.list._id;
-      userService.notify(req);
```

Comment:
The hidUsers controller calls getUsers, which populates the list. It didn't seem necessary to be going round in so many loops between the list and the user service and controller just to populate the list. But again there may be nuances I'm missing.